packages/mux-uploader/src/templates/mux-uploader.ts

```diff
@@ -0,0 +1,41 @@
+import { document } from 'shared-polyfills';
+
+import '../mux-uploader-file-select';
+import { fileSelectFragment } from '../mux-uploader-file-select';
+
+const styles = `
+:host {
+  font-family: var(--uploader-font-family, Arial);
+  font-size: var(--uploader-font-size, 16px);
+  background-color: var(--uploader-background-color, inherit);
+}
+
+input[type="file"] {
+  display: none;
+}
+`;
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+  ${styles}
+</style>
+
+<input id="hidden-file-input" type="file" />
+<mux-uploader-sr-text></mux-uploader-sr-text>
+
+<mux-uploader-status></mux-uploader-status>
+<mux-uploader-retry></mux-uploader-retry>
+
+<mux-uploader-file-select>
+  <slot name="file-select">
+    ${fileSelectFragment}
```

Comment:
I'm not following why the template for the file-select needs to be added in here as well.

Reply:
@gkatsev if we didn't do this, the `file-select` slot would still be slotted but it would have empty contents

Reply:
Ah, right. Because a `<slot>` element was slotted, which tells the WC to not use the default stuff.